.ci/appveyor.yml

```diff
@@ -49,7 +49,8 @@ install:
   - "python --version"
   - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
   - "%CMD_IN_ENV% python -m pip install --upgrade setuptools==21"
-  - "%CMD_IN_ENV% python -m pip install -r requirements.txt \
+  - "%CMD_IN_ENV% python -m pip install --install-option=--no-cython-compile \
```

Comment:
This is a potentially useful part of this PR, but the objective of this PR has been done in https://github.com/coala/coala-bears/pull/2910 , which uses a slightly different approach of first installing lxml and tries very hard to avoid letting pip select source tarballs - requiring wheels.